tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -442,6 +443,32 @@ Then('the collaborators list for file/folder/resource {string} should be empty',
   assert.strictEqual(count, 0, `Expected to have no collaborators for '${resource}', Found: ${count}`)
 })
 
-Then('the file/folder/resource {string} on the webUI should be in {string} state', function (filename, status) {
-  return client.page.sharedWithMePage().assertFileStatusIsShown(filename, status)
+Then('the file/folder/resource {string} should be in {string} state on the webUI', function (filename, status) {
+  status = status === 'Accepted' ? '' : status
+  return client.page.sharedWithMePage().assertDesiredStatusIsPresent(filename, status)
+})
+
+Given('the setting {string} in the {string} section has been set to {string}', function (setting, app, value) {
```

Comment:
similar step already exists in general Context https://github.com/owncloud/phoenix/blob/ce13f38347bc950ad144eba31cfba5c1205f9ab7/tests/acceptance/stepDefinitions/generalContext.js#L80

Please use that one